book/src/external-attacher.md

```diff
@@ -6,10 +6,11 @@
 
 **Status:** GA/Stable
 
-Latests stable release | Branch | Compatible with CSI Version | Container Image | Min k8s Version | Max k8s version
+Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | Min k8s Version | Max k8s version
 --|--|--|--|--|--
```

Comment:
Need another |-- ?

Reply:
Is Max CSI version or Max k8s version basically redundant fields? I do not see value in filling those out.

Reply:
In particular Max. CSI version means there was a breaking change and this sidecar is no longer compatible with that version of CSI. Is there a chance of this happening in post 1.0 CSI world?

Reply:
There were two reasons I had in mind for a Max CSI version field:
1. CSI spec has a 2.0
2. CSI spec introduced some new features that we haven't implemented support for yet.  As a hypothetical example, let's say external-provisioner needed to add support for resizing in 1.1, but we didn't implement it it yet, but still released a sidecar version. The max would reflect that we're not using the capabilities of the new spec yet.